### DISCUSSION TITLE
Dependency Graph not showing any dependents when there should be

### DISCUSSION THREAD

**Discussion by li-jin-gou** [community/community#44986] [Repositories]
### Select Topic Area

Bug

### Body

My repo is only showing dependencies but not dependents. Other repos show in their dependency referencing my repo but it is not showing up as a dependent on mine.

Here is my repo expecting dependents: https://github.com/cloudwego/hertz/network/dependents

similar to https://github.com/community/community/discussions/34687

It has been going on for half a year, I hope I can get help, thank you very much!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @li-jin-gou ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by li-jin-gou** [reply]
> Thanks for posting in the GitHub Community, @li-jin-gou !
> 
> We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

Thank you for your help and I am very much looking forward to the project being able to display the Dependency Graph properly.❤️

**Comment by li-jin-gou** [comment]
Hello @queenofcorgis  any progress❤️ ? Thanks very much.

**Reply by queenofcorgis** [reply]
Hi there! I did open an Issue internally with the dependency graph team - I have no timeline of when or if this will be resolved so appreciate your patience. Cheers!

**Reply by li-jin-gou** [reply]
> Hi there! I did open an Issue internally with the dependency graph team - I have no timeline of when or if this will be resolved so appreciate your patience. Cheers!

thanks ❤️

**Comment by fabiocaccamo** [comment]
Same problem here. I'm experiencing this issue on [all my Open-Source repositories](https://github.com/fabiocaccamo?tab=repositories&q=&type=source&language=&sort=stargazers).

For example these two libraries of mine, they are widely used, but the dependents list is empty:
- https://github.com/fabiocaccamo/django-admin-interface/network/dependents
- https://github.com/fabiocaccamo/python-benedict/network/dependents
